tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage The Mix manifest does not exist.
+     */
+    public function testMixMethodThrowAnExceptionIfMixManifestDoesNotExist()
+    {
+        $this->getMix()->mix('foo.css');
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage Unable to locate Mix file: /baz.css. Please check your webpack.mix.js output paths and try again.
+     */
+    public function testMixMethodThrowAnExceptionIfPathDoesNotExistInManifest()
+    {
+        $this->getMix()->mix('baz.css', 'fixtures');
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage The Mix manifest isn't a proper json file.
+     */
+    public function testMixMethodThrowAnExceptionIfManifestIsNotAProperJson()
+    {
+        $this->getMix()->setManifestFilename('mix-manifest-wrong')->mix('foo.css', 'fixtures');
+    }
+
+    public function testMixMethodWhenDisabled()
+    {
+        $this->assertEquals(new HTMLString('Mix is disabled!'), $this->getMix()->disable()->mix('foo.css'));
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     */
```

Comment:
No, not at all! 
This exception message doesn't exists, we are not expecting any exception, just check the mix method is `enabled`.
Please read the code instead of commenting every line I'm writing!